src/status_im2/contexts/communities/menus/request_to_join/view.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im2.contexts.communities.menus.request-to-join.view
   (:require [quo.react-native :as rn]
             [quo2.core :as quo]
+            [quo.components.safe-area :as safe-area]
```

Comment:
quo shouldn't be used in new designs, we have safe-area in react-native namespace

Reply:
fixed!